gettsim/transfers/wohngeld.py

```diff
@@ -39,22 +39,25 @@ def wohngeld_m_hh(
     return out
 
 
-def wohngeld_abzüge_m_tu(
+def wohngeld_abzüge_st_sozialv_m(
     eink_st_tu: float,
-    ges_rentenv_beitr_m_tu: float,
-    ges_krankenv_beitr_m_tu: float,
+    ges_rentenv_beitr_m: float,
+    ges_krankenv_beitr_m: float,
     wohngeld_params: dict,
 ) -> float:
-    """Calculate housing benefit subtractions per tax unit.
+    """Calculate housing benefit subtractions on the individual level.
+
+    Note that eink_st_tu is used as an approximation, since Lohnsteuer
```

Comment:
Not sure about this. Lohnsteuer would apply to employees only, but we'll need to think of self-employed and (eventually) civil servants, too (might well be that a low-ranked police officer in München with a large family is eligible).

Reply:
You are right, that does not seem correct, when looking at the discussion [here](https://gettsim.zulipchat.com/#narrow/stream/243798-Transfers/topic/Wohngeld/near/297925681). I will try to rephrase it, but I'm a bit unsure whether I understood it fully